extension/chrome/dev/ci_unit_test.ts

```diff
@@ -38,9 +38,10 @@ const libs: unknown[] = [
   Catch,
   Gmail,
 ];
-
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Do you think it may be worth trying to go this direction (adding properties to `Window` interface)?
https://stackoverflow.com/a/12709880

Reply:
yeah, it's worth it.
But for this line, we can't implement like that.

> (window as any)[(lib as any).name] = lib

Reply:
I think (window as any) is enough for this logic